tests/integration/test_pipeline.py

```diff
@@ -320,3 +356,125 @@ def test_concatenations_when_csv_contains_endpoints_and_resources(tmp_path):
 
     assert "i-field3" in concats["o-field2"]["fields"]
     assert "i-field4" in concats["o-field2"]["fields"]
+
+
+def test_lookups_load_csv_success(
+    pipeline_dir,
+):
+    """
+    test csv loading functionality
+    :param pipeline_dir:
+    :return:
+    """
+    lookups = Lookups(pipeline_dir)
+    lookups.load_csv()
+
+    expected_num_entries = 1
+    assert len(lookups.entries) == expected_num_entries
+
+
+def test_lookups_load_csv_failure(
+    empty_pipeline_dir,
+):
+    """
+    test csv loading functionality when file absent
+    :param pipeline_dir:
+    :return:
+    """
+    lookups = Lookups(empty_pipeline_dir)
+
+    with pytest.raises(FileNotFoundError) as fnf_err:
+        lookups.load_csv()
+
+    expected_exception = "No such file or directory"
+    assert expected_exception in str(fnf_err.value)
+
+
```

Comment:
duplication of functionality in unit test

Reply:
This test isn't in the unit test anymore.
Is this an old comment?

Reply:
It appears above the line I commented on for whatever reason. I was talking about 
`test_lookups_get_max_entity_success`
My comment below was for 
`test_lookups_add_entry_success`
which I believe still has a unit test. My concern is that they're not testing anything here in addition  to what is done by load csv or in the unit tests. Unit tests are what we should aim for.

Reply:
Hehe.
This bit's got me confused.
I'll wait to see if I've addressed the correct 'above' concern.